src/models/Rcs.ts

```diff
@@ -56,6 +56,11 @@ export class AMTConfiguration {
   wifiConfigs?: ProfileWifiConfigs[]
 }
 
+export class ProfilesWithCnt {
```

Comment:
spell out ProfilesWithCount

Reply:
Perhaps this should be `DataWithCount` and the property changed from `profiles`  to `data` to make it more generic and reusable/

Reply:
done